content/docs/command-reference/pull.md

```diff
@@ -107,89 +107,147 @@ reflinks or hardlinks to put it in the workspace without copying. See
 
 ## Examples
 
-For using the `dvc pull` command, a remote storage must be defined. (See
-`dvc remote add`.) For an existing <abbr>project</abbr>, remotes are usually
-already set up and you can use `dvc remote list` to check them. To remember how
-it's done, and set a context for the example, let's define a default SSH remote:
+Let's employ a simple <abbr>workspace</abbr> with some data, code, ML models,
+pipeline stages, such as the <abbr>DVC project</abbr> created for the
+[Get Started](/doc/tutorials/get-started). Then we can see what happens with
+`dvc pull`.
+
+<details>
+
+### Click and expand to setup the project
+
+Start by cloning our example repo if you don't already have it:
 
 ```dvc
-$ dvc remote add -d r1 ssh://_username_@_host_/path/to/dvc/remote/storage
-$ dvc remote list
-r1	ssh://_username_@_host_/path/to/dvc/remote/storage
+$ git clone https://github.com/iterative/example-get-started
+$ cd example-get-started
 ```
 
-> DVC supports several
-> [remote types](/doc/command-reference/remote/add#supported-storage-types).
+</details>
 
-Having some images and other files in remote storage, we can pull all changed
-files from the current Git branch:
+The workspace looks almost like in this
+[pipeline setup](/doc/tutorials/pipelines):
 
 ```dvc
-$ dvc pull --remote r1
+.
+├── data
+│   └── data.xml.dvc
+├── evaluate.dvc
+├── featurize.dvc
+├── prepare.dvc
+├── train.dvc
+└── src
+│   └── ...
+└── train.dvc
+```
+
+We can now just run `dvc pull` to download the most recent `model.pkl`,
+`data.xml`, and other DVC-tracked files into our local <abbr>cache</abbr> and
```

Comment:
```suggestion
We can now just run `dvc pull` to download the most recent `data/data.xml`,
`model.pkl`, and other DVC-tracked files into the <abbr>workspace</abbr> and
```
- Correct data path
- List raw data before resulting model

Reply:
> download... into our local cache and workspace

This is not exactly correct, it sounds like the data is downloaded twice or copies made in the workspace. Can you clarify the behavior please? Ping me if unsure

Reply:
yeah actually it doesn't explain correctly. I took this from pull [definition](https://dvc.org/doc/command-reference/pull#pull). 
Data is downloaded to cache and it is not copied in workspace but instead [reflinks](https://dvc.org/doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache) are used.

Reply:
Good catch. Hmmm yeah we try to keep things simple in those short introductions for cmd refs. I guess it's not incorrect after all, just lacks detail. That's OK, it's explained later in the description.

In this example let's just say "into the workspace". Mentioning the cache seems unnecessary actually.

Reply:
I'll do it.